server/.env

```diff
@@ -10,4 +10,8 @@ MQTT_HOST=
 MQTT_PORT=
 MQTT_PASSWORD=
 MQTT_USERNAME=
-MQTT_PROTOCOL= 
\ No newline at end of file
+MQTT_PROTOCOL= 
+
+# Hack to enable keycloak with self signed certs
+# don't do it in production
+NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
Typically this can be pushed without people knowing it. We had that in readme before